workshop/single-cell-resources.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Resources for single-cell RNA-seq analysis
+nav_title: Single-cell resources
+---
+
+This list provides some links to resources on single-cell RNA-seq analysis methods that may be useful to you as you develop your own single-cell RNA-seq analysis skills and practices.
+Please note, this is not an exhaustive list.
+It includes multiple types of resources for various topics in single-cell RNA-seq analysis, but does not represent the complete breadth of analysis topics.
+Resources are listed by topic and in alphabetical order, not in order of recommendation.
+
+**Table of contents**
```

Comment:
Do we want to keep the doctoc comments here for future updates, as in - https://github.com/AlexsLemonade/training-specific-template/issues/98

Reply:
Still wondering about this comment..?

Reply:
Ah sorry I meant to address this, I'll add in doctoc now.